studybuddies/src/app/tutor/create-class-tutor/create-class-tutor.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CreateClassTutorComponent } from './create-class-tutor.component';
+
+describe('CreateClassTutorComponent', () => {
+  let component: CreateClassTutorComponent;
+  let fixture: ComponentFixture<CreateClassTutorComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ CreateClassTutorComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CreateClassTutorComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7155411)
```suggestion
  it("should create", () => {
```